app/api/commands_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommandsController < Grape::API
+  namespace :commands do
+    desc 'create a new todo list'
+    params do
+      requires :name, type: String
+      optional :metadata, type: Hash, desc: 'metadata associated with the event', default: {}
+    end
+    post :create_list do
+      command = Commands::TodoList::Create.new(name: params[:name], metadata: params[:metadata])
+      if command.valid?
+        command.call
+      else
+        error!({ errors: command.errors.full_messages }, 400)
```

Comment:
A simple `post /api/commands/create_list` endpoint. This could get big and makes me wonder if we want to invest in a graphql api up front.